packages/node/src/edge-functions/edge-handler.mts

```diff
@@ -104,7 +99,7 @@ async function compileUserCode(
       ${edgeHandlerTemplate};
       const dependencies = { Request, Response };
       const options = { isMiddleware, entrypointLabel };
-      registerFetchListener(userEdgeHandler, options, dependencies);
+      registerFetchListener(handler, options, dependencies);
```

Comment:
<!-- Suggestion comments propose improvements to the code. Mention specifically what should be changed and what the benefit would be.  -->
**suggestion (non-blocking):** ‏rename `handler` to something like `userModule`

Calling this `handler` here, but `module` inside the function is confusing to follow. Using `module` as a variable name is also a bit confusing because it shadows the node module global.

Can we rename `handler` outside the function and `module` inside the function both to something like `userModule` to make this clearer?

Reply:
`userModule` sounds perfect to me!